chromedriver_autoinstaller/utils.py

```diff
@@ -222,9 +222,10 @@ def get_matched_chromedriver_version(chrome_version, no_ssl=False):
         version_url = "googlechromelabs.github.io/chrome-for-testing/known-good-versions.json"
         version_url = "http://" + version_url if no_ssl else "https://" + version_url
         good_version_list = json.load(urllib.request.urlopen(version_url))
+        compare_version = chrome_version.rsplit('.', 1)[0]
```

Comment:
Suggest appending dot to check version match. (`MAJOR.MINOR.BUILD.`)
```suggestion
        compare_version = chrome_version.rsplit('.', 1)[0] + '.'
```